api/proto/info/info.proto

```diff
@@ -347,6 +347,14 @@ message ZInfoDevice {
 
   // Information about hardware capabilities
   Capabilities capabilities = 42;
+
+  // BaseOsUpgrade Retry Counter. This must be updated only when:
+  // 1) if the configured BaseOs partition is set to UPDATED, mirror
+  // the current value of baseOs_upgrade.counter
+  // 2) At the start if a BaseOs upgrade (either from a partition in error state
+  // or from UPDATED state of another version), copy over current d
+  // deviceConfig.baseOs_upgrade_counter
+  uint32 baseOs_upgrade_counter = 43;
```

Comment:
Does this need to be per content_tree_uuid to mirror the config?
I don't know if we should report a single tuple <uuid, retry_counter> or if we should have the API have a "repeated" so we can report for at least both partitions?

Reply:
Based on our discussion a single counter is sufficient.